bootstrap.d/dev-libs.y4.yml

```diff
@@ -581,32 +581,31 @@ packages:
       categories: ['dev-libs']
     source:
       subdir: 'ports'
-      hg: 'https://gmplib.org/repo/gmp-6.2/'
-      tag: 'tip'
-      version: '6.2.1'
+      url: 'https://ftp.gnu.org/gnu/gmp/gmp-6.3.0.tar.xz'
+      format: 'tar.xz'
+      checksum: 'blake2b:a865129e2b3f634ec5bad7f97ed89532e43f5662ac47a7d8ab7f0df8c9f8d0886bd984651422e2573c2163bca69c0547c248147ec90880accbd53db97dc0ddee'
+      extract_path: 'gmp-6.3.0'
+      version: '6.3.0'
```

Comment:
Why did we switch from VCS to tar.xz?

Reply:
GMP repos are returning 403 forbidden while trying to fetch.